README.md

```diff
@@ -220,6 +220,25 @@ A volume can also be used to map this directory to one external to the container
   docker run -d -p 8080:80 -p 8021:21 -e "GALAXY_LOGGING=full" -v `pwd`/gx_logs:/home/galaxy/logs bgruening/galaxy-stable
   ```
 
+Integrating tools non-Tool Shed tools into the container
+--------------------------------------------------------
+
+We recommend to use the [Main Galaxy Tool Shed](https://toolshed.g2.bx.psu.edu/) for all your tools and workflows that you would like to share.
+In rare situations where you cannot share your tools but still want to include them into your Galaxy Docker instance, please follow the next steps.
+
+ * Get your tools into the container.
+
+    You can copy your tools into a separate folder under `/home/user/galaxy_storage/` let's say `/home/user/galaxy_storage/local_tools`, or you can mount your tool directory into the container with a separate `-v /home/user/my_galaxy_tools/:/local_tools`.
+
+ * Create a `tool_conf.xml` file for your tools. 
+ 
+    This should look similar to the main [`tool_conf.xml`](https://github.com/galaxyproject/galaxy/blob/dev/config/tool_conf.xml.sample) file, but references your tools from the new directory. In other words a tool entry should look like this `<tool file="/local_tools/application_foo/foo.xml" />`.
+    This file can be located in `/local_tools` or `/export/local_tools` depending on your setup. The only important part is, it should be available from inside of the container. We assume you have it stored under `/local_tools/my_tools.xml`.
+
+* Add the new tool config file to the Galaxy configuration.
+
+    To make Galaxy aware of your new tool configuration file you need to add the path to `tool_config_file`, which is by default `#tool_config_file = config/tool_conf.xml,config/shed_tool_conf.xml`. You can do this during container start by setting the environment variable `-e GALAXY_CONFIG_TOOL_CONFIG_FILE = config/tool_conf.xml,config/shed_tool_conf.xml,/local_tools/my_tools.xml`.
```

Comment:
The extra space around `=` after `GALAXY_CONFIG_TOOL_CONFIG_FILE` have to be removed. And `.sample` must be added as extension for `config/tool_conf.xml` and `config/shed_tool_conf.xml` (otherwise, `config/tool_conf.xml` and `config/shed_tool_conf.xml` are not found)